lib/helpers/query/handleImmutable.js

```diff
@@ -2,7 +2,7 @@
 
 const StrictModeError = require('../../error/strict');
 
-module.exports = function handleImmutable(schematype, strict, obj, key, fullPath, ctx) {
+module.exports = function handleImmutable(schematype, strict, obj, key, fullPath, options, ctx) {
```

Comment:
maybe consider adding some documentation to this function in the future

Reply:
Good suggestion, fixed in 388c5b2